src/content/guides/caching.md

```diff
@@ -148,68 +151,64 @@ __webpack.config.js__
 Let's run another build to see the extracted `manifest` bundle:
 
 ``` bash
-Hash: 80552632979856ddab34
-Version: webpack 3.3.0
+Hash: 399dc1f0f9447b1da8c0
+Version: webpack 4.7.0
 Time: 1512ms
                            Asset       Size  Chunks                    Chunk Names
-    main.5ec8e954e32d66dee1aa.js     542 kB       0  [emitted]  [big]  main
-manifest.719796322be98041fff2.js    5.82 kB       1  [emitted]         manifest
+    main.35a559f7b9f735c2373a.js     542 kB       0  [emitted]         main
+manifest.a6e25d9715318ea4928a.js    5.82 kB       1  [emitted]         manifest
                       index.html  275 bytes          [emitted]
    [0] ./src/index.js 336 bytes {0} [built]
    [2] (webpack)/buildin/global.js 509 bytes {0} [built]
    [3] (webpack)/buildin/module.js 517 bytes {0} [built]
     + 1 hidden module
 ```
 
-It's also good practice to extract third-party libraries, such as `lodash` or `react`, to a separate `vendor` chunk as they are less likely to change than our local source code. This step will allow clients to request even less from the server to stay up to date. This can be done by using a combination of a new `entry` point along with another `CommonsChunkPlugin` instance:
+It's also good practice to extract third-party libraries, such as `lodash` or `react`, to a separate `vendor` chunk as they are less likely to change than our local source code. This step will allow clients to request even less from the server to stay up to date. This can be done by using `SplitChunksPlugin` via `optimization.splitChunks` and specifiying the `chunks` to be `initial` -This is one of many option you can discover from the [`Plugin`](/plugins/split-chunks-plugin) documentation -
 
 __webpack.config.js__
 
 ``` diff
   var path = require('path');
-  const webpack = require('webpack');
   const CleanWebpackPlugin = require('clean-webpack-plugin');
   const HtmlWebpackPlugin = require('html-webpack-plugin');
 
   module.exports = {
--   entry: './src/index.js',
-+   entry: {
-+     main: './src/index.js',
-+     vendor: [
-+       'lodash'
-+     ]
-+   },
+    entry: './src/index.js',
     plugins: [
       new CleanWebpackPlugin(['dist']),
       new HtmlWebpackPlugin({
         title: 'Caching'
-      }),
-+     new webpack.optimize.CommonsChunkPlugin({
-+       name: 'vendor'
-+     }),
-      new webpack.optimize.CommonsChunkPlugin({
-        name: 'manifest'
       })
     ],
+    optimization: {
++    splitChunks : {
++      chunks: 'initial'
++    },
+     runtimeChunk : {
+       name: 'manifest'
+     }
+    }, 
     output: {
       filename: '[name].[chunkhash].js',
       path: path.resolve(__dirname, 'dist')
     }
   };
 ```
 
-W> Note that order matters here. The `'vendor'` instance of the `CommonsChunkPlugin` must be included prior to the `'manifest'` instance.
+T> Note that order matters here. The order of `'splitChunks'` and `'runtimeChunk'` doesn't have any importance in this context.
```

Comment:
These two sentences seem to contradict each other.

Reply:
Yes, since last version of CommonChunkPlugin was depending on the order and now with SplitChunksPlugin there's no need for an order, so i thought of adding it a Tip, notice that i've deleted the Warning and replaced it with a Tip.

Reply:
I would remove this paragraph altogether.

Reply:
Old documentation over over articles in older version states that the order is needed, don't you think it might rise some questioning ?

Reply:
No, I think this is information is irrelevant now and will only confuse people.